composeApp/src/main/kotlin/io/github/couchtracker/tmdb/TmdbDownload.kt

```diff
@@ -100,7 +95,7 @@ suspend fun <T> tmdbDownloadResult(
     return runApiCatching(logTag) {
         try {
             injectApiError()
-            val client = tmdb
+            val client = KoinPlatform.getKoin().get<Tmdb3>()
```

Comment:
Not for this PR, but the looking at this again the real issue is that we're overusing top-level functions. This doesn't easily allow for proper dependency injection and makes code harder to test and harder to integrate with Koin.

Ideally, these functions would be in a `TmdbDownloader` class, that can be declared as a singleton via Koin.